osu.Game/Screens/OnlinePlay/Lounge/Components/RoomPublicityFilter.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+namespace osu.Game.Screens.OnlinePlay.Lounge.Components
+{
+    public enum RoomPublicityFilter
+    {
+        Public,
+        Private
+    }
```

Comment:
I think we want an `All` option back here and set to default as well. The user should be able to see all rooms regardless of accessibility by default.